pkg/promotion/runner/builtin/git_pr_merger.go

```diff
@@ -134,12 +184,27 @@ func (g *gitPRMerger) run(
 		if merged {
 			break
 		}
-		if i < maxMergeAttempts {
+		if i < maxMergeAttempts-1 {
```

Comment:
Why was this added? The `range` up above is an exclusive range, so if the number is 10, then it does 0-9